File: POOBanca/PHP/Hq.php
<?php
require_once "Revista.php";

class Hq extends Revista {
    public function __construct($nome, $editora, $totPaginas, $numero) {
        parent::__construct($nome, $editora, $totPaginas, $numero);
    }

    public function __destruct() {
        echo "<p>Objeto Hq Destruído com Sucesso!</p>";
    }
}

File: POOBanca/C++/Hq.cpp
#include "Hq.h"

Hq::Hq(string nome, string editora, int totPaginas, int numero) : Revista(nome, editora, totPaginas, numero) {

}

Hq::~Hq() {
    cout << "Objeto Hq Destruído com Sucesso!" << endl;
}
